core/modules/widgets/view.js

```diff
@@ -18,19 +18,95 @@ var ViewWidget = function(parseTreeNode,options) {
 	this.initialise(parseTreeNode,options);
 };
 
+var ViewHandler = function(parseTreeNode,options) {
+	this.initialise(parseTreeNode,options);
+};
+
 /*
 Inherit from the base widget class
 */
 ViewWidget.prototype = new Widget();
 
+/*
+Inherit from the base widget class
+*/
+ViewHandler.prototype = new Widget();
+
+/*
+Render this widget into the DOM
+*/
+ViewHandler.prototype.render = function(parent,nextSibling,options) {
```

Comment:
Using `options` here is a problem if the function is called without the options parameter, which is common, because "options" is optional. 

So we need to initialize it -> otherwise line 42ff will probably throw RSODs

`options = options || {}`   without any var

Reply:
I've changed the code.